DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/SimpleInjectorConfiguration.cs

```diff
@@ -28,7 +20,8 @@ public static Container RegisterPackages(Container container)
         {
             container.RegisterPackages(new[]
             {
-                typeof(DALPackage).Assembly,
+                typeof(WebApiActivityPackage).Assembly,
+                typeof(DALPackage).Assembly, 
```

Comment:
my package named WebApiActivityPackage,  DALPackage is package from security, it isn't informative and it will be renamed to SecurityDALPackage